compiler/ballerina-parser/src/main/java/io/ballerinalang/compiler/internal/parser/BallerinaParserErrorHandler.java

```diff
@@ -1346,6 +1348,7 @@ protected ParserRuleContext getNextRule(ParserRuleContext currentCtx, int nextLo
             case KEY_SPECIFIER:
             case ERROR_TYPE_DESCRIPTOR:
             case LET_VAR_DECL:
+            case ERROR_CONSTRUCTOR_EXPRESSION:
```

Comment:
Even-though a context is started here, its never ended.

Should be ended after ending the args-list context.

Reply:
Removed this context.